featuretools/computational_backends/calculate_feature_matrix.py

```diff
@@ -444,33 +444,31 @@ def approximate_features(features, cutoff_time, window, entityset, backend,
     for approx_entity_id, approx_features in to_approximate.items():
         # Gather associated instance_ids from the approximate entity
         cutoffs_with_approx_e_ids = approx_cutoffs.copy()
-        frames = entityset.get_pandas_data_slice([approx_entity_id, target_entity.id],
-                                                 target_entity.id,
-                                                 cutoffs_with_approx_e_ids[target_instance_colname])
-
-        if frames is not None:
```

Comment:
I'm not sure what this is trying to check, and the `else` branch was untested, so I just got rid of it.

Reply:
I added a test to master that hits the `else` branch